### PR TITLE
Expose profile id and match JavaScript sdk event interface

### PR DESCRIPTION
**Pull request by tom-quiltt**
Close: #10 
Close: https://github.com/quiltt/quiltt-connector-flutter-sdk/issues/4

### CODE REVIEW

File: lib/event.dart [hunk 1]
```diff
@@ -1,21 +1,57 @@
-class EventMetadata {
+class ConnectorSDKCallbackMetadata {
   String connectorId;
   String? connectionId;
   String? moveId;
```
Comment (tom-quiltt) [inline lib/event.dart:4]: I noticed `moveId` isn't present in [ConnectorSDKCallbackMetadata in browser.ts](https://github.com/quiltt/quiltt-public/blob/abc04d8de57b448dddbe4b0656aa5f57bd0b7d7c/ECMAScript/core/src/api/browser.ts#L68), why is that? @sirwolfgang

Comment (sirwolfgang) [reply lib/event.dart:4]: Because nothing returns `moveId`. Idk what that's from.

Comment (tom-quiltt) [reply lib/event.dart:4]: ok, probably a miscommunication on `moveId`, removing.

**Comment by tom-quiltt** [comment]
@sirwolfgang just wanna make sure, there will not be `moveId` after the move module, I remove `moveId` in https://github.com/quiltt/quiltt-connector-flutter-sdk/pull/12/commits/3458e49f9cdb5fe2cc32424a292e1bdc46d411c8.

**Pull request merged**